cordyceps/cordyceps/path_planner.py

```diff
@@ -21,7 +21,7 @@ def __init__(self):
 
     def generate_vs_path_mock(self, start_pose:Pose) -> np.array:
 
-        file = open('/home/mart/Documents/git/Turtlebot3_Simulation_WorkSpace/turtle_ws/src/Cordyceps/cordyceps/resource/sine_big.csv','r')
+        file = open('/home/tangouniform/Documents/cordycepsws/Turtlebot3_Simulation_WorkSpace/turtle_ws/src/Cordyceps/cordyceps/resource/Path1.csv','r')
```

Comment:
use relative path as it doesn't have to be changed everytime
```suggestion
        file = open('cordyceps/resource/Path1.csv','r')
```